INTITA/Modules/LogInScreen/LogInViewController.swift

```diff
@@ -80,7 +81,7 @@ class LogInViewController: UIViewController, Storyboarded, AlertAcceptable {
         tableViewBottomContraint.constant = height
         view.layoutSubviews()
         
-        tableView.scrollToRow(at: IndexPath(row: 5, section: 0), at: .bottom, animated: true)
+        tableView.scrollToRow(at: IndexPath(row: 7, section: 0), at: .bottom, animated: true)
```

Comment:
Same story.` LoginCells.loginButtonCell.rowIndex`

Reply:
I have just done it.